pkg/inst/tests/test_sparkSQL.R

```diff
@@ -220,4 +220,31 @@ test_that("sampleDF on a DataFrame", {
   expect_true(count(sampled2) < 3)
 })
 
+test_that("select with column", {
+  df <- jsonFile(sqlCtx, jsonPath)
+  df1 <- select(df, "name")
+  expect_true(columns(df1) == c("name"))
+  expect_true(count(df1) == 3)
+
+  df2 <- select(df, df$age)
+  expect_true(columns(df2) == c("age"))
+  expect_true(count(df2) == 3)
+})
+
+test_that("column calculation", {
+  df <- jsonFile(sqlCtx, jsonPath)
+  d <- collect(select(df, alias(df$age + 1, "age2")))
+  expect_true(names(d) == c("age2"))
+  df2 <- select(df, lower(df$name), abs(df$age))
+  expect_true(inherits(df2, "DataFrame"))
+  expect_true(count(df2) == 3)
+})
+
+test_that("column operators", {
+  c <- col("a")
```

Comment:
Right now, we can not do this in Scala/Python, because DataFrame in immutable, `withColumn()` will return a new DataFrame.

But we can try this in R, replace the underline `sdf` with new one.